bedrock/mozorg/templates/mozorg/about/manifesto.html

```diff
@@ -275,7 +275,7 @@ <h1 itemprop="name">{{ self.page_title() }}</h1>
                   <h4>{{ _('Learn more') }}</h4>
                   <ul class="resources">
                     <li><a href="http://party.webmaker.org/">{{ _('Host or join a Maker Party') }}</a></li>
```

Comment:
Looks like this is also broken, should update to `https://teach.mozilla.org/events/`